api/controllers/getAnimeByIdController.go

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import (
+	animeU "bootCampApi/api/usecases"
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/fatih/structs"
+	"github.com/gorilla/mux"
+)
```

Comment:
imports should be ordered alphabetically and separated into three main blocks (or two, or only one, if other blocks are not presented), which are:
- Go's standard library/packages.
- internal packages.
- Third-party packages.

```suggestion
import (
	"encoding/csv"
	"fmt"
	"log"
	"net/http"
	"os"
	
	animeU "bootCampApi/api/usecases"

	"github.com/fatih/structs"
	"github.com/gorilla/mux"
)
```